lib/models/package-info-cache/package-info.js

```diff
@@ -530,10 +516,14 @@ class PackageInfo {
    * @return {Object} the constructed instance of the addon
    */
   getAddonInstance(parent, project) {
-    let ctor = this.getAddonConstructor();
+    let addonEntryPointModule = require(this.addonMainPath);
```

Comment:
`getAddonConstructor` checks to ensure that addonMainPath is present and throws a nice error, if we move to directly requiring (which seems fine) we should ensure that we do the same nice error

Reply:
Ah, good call. I'll update